isolate_test.go

```diff
@@ -39,6 +39,9 @@ func TestIsolateTermination(t *testing.T) {
 	if e := <-err; e == nil || !strings.HasPrefix(e.Error(), "ExecutionTerminated") {
 		t.Errorf("unexpected error: %v", e)
 	}
+	if iso.IsExecutionTerminating() {
```

Comment:
Should we have another test for when this returns `false`?  Should be easy to do that just after creating the isolate.

Reply:
That is checking that it returns false. We'd need a test case to vet it returns true.

Reply:
We'd need to reproduce the situation where terminating execution of some javascript isnt happening as fast as we think it is.

Reply:
In that case, we might need to do the check in a Go callback for a function template.  V8 could terminate nested JS execution, but then would need to return control back to Go code while there would still be a JS frame on the stack.